std/range/package.d

```diff
@@ -8269,6 +8269,22 @@ unittest // issue 14373
     assert(r.empty);
 }
 
+unittest // issue 14575
+{
+    struct R
+    {
+        Object front;
+        alias back = front;
+        bool empty = false;
+        void popFront() {empty = true;}
+        alias popBack = popFront;
+        @property R save() {return this;}
+    }
+    static assert(isBidirectionalRange!R);
+	R r;
+	auto rr = refRange(&r);
```

Comment:
> How is having front as a member variable "abnormal"? I do that in my own code quite often.

It's generally considered bad practice to have public member variables - rather property functions are typically used to wrap what would otherwise be a public member variable - so I would fully expect that having `front` as a public member variable would be rare. AFAIK, there is no case in Phobos where it's done, and as far as I recall, this is the first time that I've ever seen anyone post code where they've done it. Certainly, it should be supported, and the range API doesn't forbid it, but it's not something that I would expect to see normally.